doc/source/usage.rst

```diff
@@ -77,11 +78,17 @@ Here you can see the command-line usage instructions for the main looper command
 	                        exists marking the run (e.g. as 'running' or
 	                        'failed'). Set this option to ignore flags and submit
 	                        the runs anyway.
+	  --ignore-duplicate-names
```

Comment:
Is this already pretty integrated, here or in `pypiper`? If not, could we consider a name change? The description suggests something like `overwriter-existing` or `overwrite-duplicates`. `ignore...` sounds like something that _isn't_ happening, while this option sounds like it's actually to make something _happen_

Reply:
not integrated at all. feel free to change! I think you chose that name :). but what to change to?

the default behavior is: looper will NOT submit a job for a sample_name that was already present in the table. This turns that behavior off... It shoul maybe be `allow-duplicate-names` ?

Reply:
Ha that's quite possible. What about `overwrite-duplicates`?

Reply:
well... it doesn't technically overwrite anything, really, does it?  It depends on what your pipeline does... i've used it, for example, when I'm using other variables to make things unique. nothing really getting overwritten (except maybe the throwaway submission script?). so that seems to be implying something that is not always true.

Reply:
Ah OK, what about something like `force-rerun`

Reply:
well, there's no rerunning going on...all it does is *not* skip a row with an identical name.

Reply:
Ah sorry, I'd mistakenly thought this was about folder names in addition to sample names, e.g. same output folders or something. OK yeah this makes more sense then. I like `submit-repeats` but am also fine with as-is.

Reply:
well, typically it is, but that's a downstream thing, not necessarily determined by looper...that's just our pipeline convention.

how about `allow-repeats`?

Reply:
or `allow-repeat-names` ? or `allow-duplicate-names` ? submit may or may not happen depending on other things